lib/Frontend/CompilerInvocation.cpp

```diff
@@ -343,6 +343,10 @@ static bool ParseAnalyzerArgs(AnalyzerOptions &Opts, ArgList &Args,
         Success = false;
         break;
       }
+      if (key == "ctu-import-threshold") {
```

Comment:
Shouldn't we have to change the corresponding lit tests?

Reply:
Actually there no tests for the existence of analyzer options. Those will arrive with the next clang version. IMHO it would be better to leave the possibility open (on the ctu-clang7 branch) for specifying this threshold in 2 ways:
a) -analyzer-ctu-import-threshold 100
b) -analyzer-config ctu-import-threshold=100
When we uplift all of this will go away anyway.